
public class secondException {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try{
			
		}catch(ArithmeticException e)
		{
			
		}catch(RuntimeException e)
		{
			
		}catch(Exception e)
		{
			
	
		}
	}
	
}

		

	


